### DISCUSSION TITLE
Cannot install Nadine as documented and not using docker-compose

### DISCUSSION THREAD

**Discussion by jonasdagefoerde** [nadineproject/nadine#461] [Q&A] [answered]
Hi,

I want to install Nadine on OSX 10.15.7 with Docker Desktop 3.1.0. Using the [documented installation steps](https://nadine.readthedocs.io/en/master/quickstart.html) throws an error telling me `requirements.txt` file cannot be found. I am trying to install the current Github version. There is a `requirements.rst` file in the `docs` folder but I do not know if it replaces the former txt file.

I tried another way to install Nadine using `docker-compose up`. Unfortunately, this also throws an error:
```
docker-compose up
[...]
Successfully installed pip-21.0
Removing intermediate container 7292b718c3e7
 ---> 35fb08ea51ef
Step 13/17 : RUN pip install -r $DOCKYARD_SRVPROJ/requirements.txt
 ---> Running in 906d2805e263
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/webapp/nadine/requirements.txt'
ERROR: Service 'django' failed to build : The command '/bin/sh -c pip install -r $DOCKYARD_SRVPROJ/requirements.txt' returned a non-zero code: 1
```
The postgres db container starts and the db is created - although only after fixing the user password error.

This leaves me wondering how Nadine can be installed on OSX. May be I do get the process for installing wrong. Any help is appreciated :)

Cheers
Jonas

**Comment by jsayles** [comment, marked as answer]
I haven’t used docker to run Nadine in quite some time.  If you can get it working I’d love to get that updated.  I do run this on a Mac and I use pipenv not docker.
